applications/vanilla/views/categories/helper_functions.php

```diff
@@ -169,7 +169,7 @@ function writeListItem($category, $depth) {
                 Gdn::controller()->EventArguments['Category'] = &$category;
                 Gdn::controller()->fireEvent('BeforeCategoryItem');
                 $headingClass = "CategoryNameHeading";
-                if (empty($row['Description'])) {
+                if (empty($category['Description'])) {
```

Comment:
This is the only time I would vote for `val()` :-)
```suggestion
                if (empty(val('Description', $category))) {
```